okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/OkHttpClients.java

```diff
@@ -209,7 +211,11 @@ private static RemotingOkHttpClient createInternal(
         client.followRedirects(false); // We implement our own redirect logic.
 
         // SSL
-        client.sslSocketFactory(new KeepAliveSslSocketFactory(config.sslSocketFactory()), config.trustManager());
+        SSLSocketFactory sslSocketFactory = MetricRegistries.instrument(
+                config.taggedMetricRegistry(),
+                new KeepAliveSslSocketFactory(config.sslSocketFactory()),
+                serviceClass.getSimpleName());
```

Comment:
Going by how we handle this on the server side, should this just be `"client"` rather than the service class's name?

Reply:
The server uses "server" because all incoming connections are the same, we already have tags for the local service name. A single service may have many outgoing clients, so it may be helpful to see that FooService client is negotiating GCM ciphers, while BarService is not.